src/components/shared/formItems/ProjectTicketMods.tsx

```diff
@@ -16,6 +16,10 @@ import FormattedAddress from '../FormattedAddress'
 import NumberSlider from '../inputs/NumberSlider'
 import { FormItemExt } from './formItemExt'
 
+const MODAL_MODE_EDIT = 'Edit'
+const MODAL_MODE_ADD = 'Add'
+type MODAL_MODE = 'Add' | 'Edit'
```

Comment:
since a type error would be thrown using any value other than "add" or "edit", I think we can get by without defining these values as variables

also nit pick: the app uses TitleCase for type declarations, so `ModalMode` instead of `MODAL_MODE` please 😊